ocpp/charge_point.py

```diff
@@ -170,20 +170,22 @@ def _raise_key_error(action, version):
     from ocpp.v201.enums import Action as v201_Action
 
     if version == "1.6":
-        if hasattr(v16_Action, action):
+        try:
+            v16_Action(action)
             raise NotImplementedError(
```

Comment:
using try/except to determine if an error should be raised (the purpose of the function `_raise_key_error`) doesn't seem the best approach. also, try/except is used to catch _unexpected_ expected exceptions within other code wrapped inside, raising an explicit exception within the block isn't great practice.

Reply:
Note the try/except blocks are being used to catch expected _exceptions_.

Reply:
I don't think we have any other option that to check the membership using try/except clauses here.

Reply:
Resolving this. We will stick with try/except clause for now.